Userland/Libraries/LibC/time.cpp

```diff
@@ -87,6 +88,32 @@ int utimes(char const* pathname, const struct timeval times[2])
     return utime(pathname, &buf);
 }
 
+// Not in POSIX, originated in BSD but also supported on Linux.
+// https://man.netbsd.org/NetBSD-6.0/lutimes.2
+int lutimes(char const* pathname, const struct timeval tv[2])
```

Comment:
We use east-const, so `struct timeval const` instead of `const struct timeval`. `futimes` already does this correctly, but the two definitions in `time.h` need an update as well.

Reply:
Thanks!